src/auth-service/utils/create-maintenance.js

```
@@ -0,0 +1,133 @@
+
+      const filter = generateFilter.maintenances(request, next);
+      const responseFromRemoveMaintenance = await MaintenanceModel(
+        tenant
+      ).remove(
+        {
+          filter,
+        },
+        next
+      );
+      return responseFromRemoveMaintenance;
+    } catch (error) {
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)